PubHub/PubHub.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PubHub.API.Domain;
+using PubHub.Common.Models;
+
+namespace PubHub.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class UserController(PubHubContext context) : Controller
+    {
+        private readonly PubHubContext _context = context;
+
+        /// <summary>
+        /// Get all general information about a specific user.
+        /// </summary>
+        /// <param name="id">ID of user.</param>
+        /// <response code="200">Success. User information was retreived.</response>
+        /// <response code="404">The user wasn't found.</response>
+        /// <response code="500">Unexpected error.</response>
+        [HttpGet("{id}", Name = "GetUser")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public IActionResult GetUser(int id)
+        {
+            var userModel = _context.PubHubUsers
+                .Include(u => u.Account)
+                .ThenInclude(a => a!.AccountType)
+                .Select(u => u.MapUserModelProperties())
+                .SingleOrDefault(u => u.Id == id);
+            if (userModel == null)
+            {
+                return NotFound($"No user with ID: {id}");
```

Comment:
Should we not use [RFC9110](https://datatracker.ietf.org/doc/html/rfc9110) as error response standard?